### PR TITLE
Initial release (pa_ppx_string)

### PR DESCRIPTION
**Pull request by chetmurthy**
A PPX rewriter like ppx_string, but based on Camlp5 so compatible with all the other Camlp5-based rewriters.

### CODE REVIEW

**Comment by chetmurthy** [comment]
fixed minor bug

**Comment by chetmurthy** [comment]
Looks like every thing passed except for macos-homebrew -- which fails b/c of lack of some conf-perl-* packages.  Can  you release this package ?

File: packages/pa_ppx_string/pa_ppx_string.0.01/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+maintainer: "Chet Murthy <chetsky@gmail.com>"
+authors: ["Chet Murthy"]
+homepage: "https://github.com/camlp5/pa_ppx_string"
+license: "BSD-3-Clause"
+bug-reports: "https://github.com/camlp5/pa_ppx_string/issues"
+dev-repo: "git+https://github.com/camlp5/pa_ppx_string.git"
+doc: "https://github.com/camlp5/pa_ppx_string/doc"
+
+depends: [
+  "ocaml"       { >= "4.10.0" & < "5.01.0" }
```
Comment (mseri) [inline packages/pa_ppx_string/pa_ppx_string.0.01/opam:18]:
```suggestion
  "ocaml"       { >= "4.10.0" & < "5.1.0" }
```
Is the upper bound on "5.1" really needed? We could add it later if something breaks

Comment (chetmurthy) [reply packages/pa_ppx_string/pa_ppx_string.0.01/opam]: Good point, removed the upper bound.

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**